src/config/2022.json

```diff
@@ -479,7 +478,8 @@
       ],
       "avatar_url": "1098110",
       "github": "ColinEberhardt",
-      "website": "http://www.scottlogic.com"
+      "website": "https://blog.scottlogic.com/ceberhardt/",
+      "twitter": "ColinEberhardt"
```

Comment:
Found you online looking for a bio. Feel free to edit/remove if you'd prefer not to list here.